src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar"
+      data-qa="big"
+      action="#"
+      method="get"
+    >
+      <div>
+        <img
+        class="search-bar__loupe"
+        alt="search-loupe"
+        src="./images/Search.svg"
+        >
```

Comment:
Add proper indentation
```suggestion
        <img
          class="search-bar__loupe"
          alt="search-loupe"
          src="./images/Search.svg"
        >
```